.circleci/config.yml

```diff
@@ -137,6 +137,22 @@ jobs:
           command: make -C docs
       - save_sbt_cache
 
+  samples-tests:
+    machine: true
+    description: "Shopping Cart Integration tests (Value Entity)"
+    steps:
+      - checkout
+      - setup_sbt
+      - restore_sbt_cache
+      - run:
+          name: Run integration tests
+          command: |
+            sbt publishM2
+            export SDK_VERSION=`sbt sdk/version | tail -n 1 | cut -c8-100`
+            cd samples/java-valueentity-shopping-cart 
+            mvn -Dakkaserverless-sdk.version=$SDK_VERSION clean verify -Pit
```

Comment:
Still very hacky. 

It's annoying to `publishM2` for each sample test. 

I need to test further, but an alternative is to have job that creates a `version.sbt` fixing the version to some build id from CircleCI. We run `publishM2` once and we run the other sample jobs using the same fixed version.